docs/intellij-automated-setup-behavior.md

```diff
@@ -22,7 +22,7 @@ The syntax for the pre-configured settings found below are as follows:
 | IntelliJ Setting                      | Node                                 | Value               |
 |---------------------------------------|--------------------------------------|---------------------|
 | `Active` `Configuration File`         | `entry[@key='active-configuration']` | `PROJECT_RELATIVE:$PRJ_DIR$/static-analysis/teammates-checkstyle.xml:teammates` |
-| `Checkstyle version:`                 | `entry[@key='checkstyle-version']`   | `8.7`               |
+| `Checkstyle version:`                 | `entry[@key='checkstyle-version']`   | `8.7`[[1]](#versions-sync-with-tools-used-in-build-script) |
```

Comment:
Can we just let the user refer the `build script` for version? Put a "hard-code" version is fine but will increase our works if we upgrade CheckStyle version (and people may forget to update the documentation here.) What do you think?

Reply:
I agree, thanks for your meticulous review!